alialiwa2005.md

```diff
@@ -109,13 +120,7 @@ Whichever path you choose to travel with this build, you can always HACK! :)
 
 ## Budget
 
-| Product                         | Supplier/Link                         | Cost    |
-| ------------------------------- | ------------------------------------- | ------- |
-| 4 Raspberry Pi 4 4GB Ram        | https://bit.ly/2ZVDImM                | $220    |
-| 4 Power Supplies                | https://bit.ly/3WwznmU                | $32     |
-| 6 Ethernet Cables               | https://bit.ly/3WtkAte                | $3.23   |
-| 4 MicroSD Cards                 | (Already Own)                         | $0      |
-| Customized Case                 | (Already Own Supplies)                | $0      |
-| Gigabit Ethernet Network Switch | (Already Own)                         | $0      |
-| Router for Project-Only Network | (Already Own a Spare Router)          | $0      |
-| Total Including NYC Tax (8.875%)|                                       | $277.89 |
+| Product                                 | Supplier/Link                         | Cost    |
+| --------------------------------------- | ------------------------------------- | ------- |
+| Libre Web Services                      | https://amzn.to/3vwx5ZI               | $277.37 |
```

Comment:
love that this is super convenient for ordering hehe but for records reasons, could you please break it out into individual components? also, is this total inclusive of estimated shipping and taxes?

thank you sm!